decentraland/package.json

```diff
@@ -38,6 +38,7 @@
     "tasit-sdk": "0.0.15"
   },
   "devDependencies": {
+    "argv": "0.0.2",
```

Comment:
It's weird that the dep is `argv` and not `yargs`

Reply:
Ops, fixed. I guess that was a wrong copy-paste.

Reply:
Ah ok, glad I checked. Thanks!

Are you not doing `npm i --save-dev` when adding deps?

Reply:
Yes, I'm doing but I guess that I've copied wrongly the name of the package. I've installed twice, the first time did ok but the second didn't. Since the package was already installed I didn't realized that it was wrong.

Reply:
Ah ok. Makes sense! Resolving this.